src/main/java/com/duke/task/TaskList.java

```diff
@@ -0,0 +1,35 @@
+package com.duke.task;
+
+import com.duke.task.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> list;
+    private int count;
```

Comment:
Maybe it would be easier to use `list.size()` instead of maintaining a variable `count`?